db/migrate/20170921065259_update_adjustment_states.spree.rb

```diff
@@ -0,0 +1,17 @@
+# This migration comes from spree (originally 20130417120035)
+class UpdateAdjustmentStates < ActiveRecord::Migration
+  def up
+    Spree::Order.complete.find_each do |order|
+      order.adjustments.update_all(:state => 'closed')
+    end
+
+    Spree::Shipment.shipped.includes(:adjustment).find_each do |shipment|
+      shipment.adjustment.update_column(:state, 'finalized') if shipment.adjustment
+    end
+
+    Spree::Adjustment.where(:state => nil).update_all(:state => 'open')
+  end
+
+  def down
+  end
+end
```

Comment:
damn! we shouldn't have missed this :see_no_evil:

Reply:
It's amazing that you only missed one! There is so much code to look through - impossible to catch everything! @enricostano found this - I looked for it after the bug appeared and I couldn't find it even though I knew what I was looking for!!! ❤️

Reply:
Note to myself. DDL migrations very very likely come with a data migration :sweat_smile: